client/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
While not mandatory, it is recommended that you include the name of your project as it can enhance search engine optimization after deployment.

Reply:
Done